app/models/hourly_rate.rb

```diff
@@ -65,7 +65,7 @@ def self.at_date_for_user_in_project(date, user_id, project = nil, include_defau
       if rate.nil?
         project = Project.find(project) unless project.is_a?(Project)
         rate = find(:first,
-                    :conditions => [ "user_id = ? and project_id in (?) and valid_from <= ?", user_id, project.ancestors, date],
+                    :conditions => [ "user_id = ? and project_id in (?) and valid_from <= ?", user_id, project.ancestors.to_a, date],
```

Comment:
Line is too long. [133/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside square brackets detected.